README.md

```diff
@@ -123,16 +123,5 @@ guide](https://docs.bazel.build/versions/3.2.0/install.html)).
 Bazel will pull pypi libraries automatically, but you need to have pip3
 installed.
 
-To run the script using Bazel, first execute `sh setup.sh`, which will grab pypi
+To run the script using Bazel, execute `bazel build //main:train_classifier_g`, which will grab pypi
```

Comment:
could change to bazel run, then say later in the sentence that it'll build and run

Reply:
I think I should keep `bazel build` here, because we are no longer running the script directly using bazel.